apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/Logging/LoggerOptions.cs

```diff
@@ -74,20 +85,24 @@ public sealed class LoggerOptions
         ///     See: https://cloud.google.com/logging/docs/api/v2/resource-list </param>
         /// <param name="bufferOptions">Optional, the buffer options.  Defaults to a <see cref="BufferType.Timed"/></param>
         /// <param name="retryOptions">Optional, the retry options.  Defaults to a <see cref="RetryType.None"/></param>
+        /// <param name="loggerDiagnosticsOutput">Optional. If set some logger diagnostics info will be written
+        /// to the given <see cref="TextWriter"/>. Currently the only diagnostics info we provide is the URL where
+        /// the logs written with these options can be found.</param>
         public static LoggerOptions Create(
             LogLevel logLevel = LogLevel.Information,
             string logName = null,
             Dictionary<string, string> labels = null,
             MonitoredResource monitoredResource = null,
             BufferOptions bufferOptions = null,
-            RetryOptions retryOptions = null)
+            RetryOptions retryOptions = null,
+            TextWriter loggerDiagnosticsOutput = null)
```

Comment:
This is a binary breaking change - am I right in saying we're okay with that at the moment, as we're still pre-3.0-GA?

Reply:
Yes, this is all pre 3.0 GA